Changelog.md

```diff
@@ -1,5 +1,91 @@
 ## Ignition Gazebo 6.x
 
+### Gazebo Sim 6.11.0 (2022-08-17)
+
+1. Add support for specifying log record period
+    * [Pull request #1636](https://github.com/gazebosim/gz-gazebo/pull/1636)
```

Comment:
All these urls need to be updated to `gz-sim`

```suggestion
    * [Pull request #1636](https://github.com/gazebosim/gz-sim/pull/1636)
```

Reply:
fixed. [2f68bc4](https://github.com/gazebosim/gz-sim/pull/1656/commits/2f68bc4253e15d0c0a89dcde8fc21342bbcd3545)